modules/standard/IO.chpl

```diff
@@ -2847,14 +2847,8 @@ proc _channel.withFormatter(f: ?) {
   return ret;
 }
 
-/*
-
-Represents a Unicode codepoint. I/O routines (such as :proc:`channel.read`
-and :proc:`channel.write`) can use arguments of this type in order to read or
-write a single Unicode codepoint.
-
- */
-record ioChar {
+pragma "no doc"
+record _internalIoChar {
   /* The codepoint value */
   var ch:int(32);
```

Comment:
My interpretation of your Slack conversation with Lydia was that the old `ioChar` using `int(32)` was an inconsistency and it would be better to now use `int(64)` for this (but this is worth double checking as I might be reading it wrong)

Reply:
Rereading the slack conversation again leaves me unclear on this particular case, as it is now an internal implementation.
@lydia-duncan can you comment on this?

Reply:
The public interface for the new function should be just `int` but this part is now internal and it looks like it is passing that value to places that expect an `int(32)`.  So I think it is okay to leave it as is (but maybe we should be safeCasting instead of just casting)